src/views/BannerSubmission/index.js

```diff
@@ -0,0 +1,85 @@
+import Typography from '@material-ui/core/Typography';
+import React, { Component } from 'react';
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+import { gordonColors } from '../../theme';
+import Version from '../../services/version';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import './banner.css';
+
+export default class BannerSubmission extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loading: true,
+      version: null,
+    };
+  }
+
+  componentWillMount() {
+    this.loadData();
+  }
+
+  async loadData() {
+    const versionPromise = Version.getVersion();
```

Comment:
I wonder whether it would be cleaner to do it this way:
```suggestion
    const version = await Version.getVersion();
```

Not mission critical, but just a suggestion